week1/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	// "errors"
+)
+
+func main() {
+
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Print(">")
+	for scanner.Scan(){
+		text := scanner.Text()
+		if text != "" {
+			fmt.Print(process(text))
+			}
+		fmt.Print("\n >")
```

Comment:
Above Print doesn't have space before `>`, so it's best to define `>` as a constant